x-pack/metricbeat/module/sql/query/query_integration_test.go

```diff
@@ -158,10 +158,15 @@ func TestPostgreSQL(t *testing.T) {
 				Assertion: func(t *testing.T, event beat.Event) {
 					value, err := event.GetValue("sql.query")
 					assert.NoError(t, err)
-					require.NotEmpty(t, value.(map[string]interface{}))
+					require.NotEmpty(t, value)
 				},
 			}
 
+			// Need to add this.
```

Comment:
Why is it needed? I'd assume that any code we add is "needed" :slightly_smiling_face: 

If this test requires any explanation, please comment it here. If it doesn't, please remove this comment.

Reply:
Removed